### PR TITLE
Set account & env

### PR DESCRIPTION
**Pull request by bunsenstraat**
fix https://github.com/ethereum/remix-project/issues/793

### CODE REVIEW

File: apps/remix-ide/src/app/tabs/runTab/settings.js [hunk 1]
```diff
@@ -319,6 +323,22 @@ class SettingsUI {
     )
   }
 
+  getSelectedAccount () {
+    return this.el.querySelector('#txorigin').selectedOptions[0].value
+  }
+
+  setSelectedAccount (account) {
```
Comment (yann300) [inline apps/remix-ide/src/app/tabs/runTab/settings.js:330]: shouldn't that require feedback to the user (like a validation step)? was that also asked by plugin developer?

Comment (bunsenstraat) [reply apps/remix-ide/src/app/tabs/runTab/settings.js:330]: ok

Comment (bunsenstraat) [reply apps/remix-ide/src/app/tabs/runTab/settings.js:330]: added permissions

File: apps/remix-ide/src/app/udapp/run-tab.js [hunk 2]
```diff
@@ -44,6 +45,61 @@ export class RunTab extends LibraryPlugin {
...
+          selectedEnvMode: this.blockchain.getProvider(),
+          networkEnvironment: this.container.querySelector('*[data-id="settingsNetworkEnv"]').textContent
+        }
+        )
+      }
+    })
+  }
+
+  async setEnvironmentMode (env) {
+    const canCall = await this.askUserPermission('setEnvironmentMode', 'change the enivornment used')
```
Comment (yann300) [inline apps/remix-ide/src/app/udapp/run-tab.js:72]: typo `enivornment`

**Review by yann300 — APPROVED** [review]

**Pull request merged**